1.Fuzzing_technology/README.md

```diff
@@ -285,13 +318,18 @@ Your task is to do the following:
 
 ## Task 4
 
-### Creating your own fuzzer and fuzzing it
+### Creating your own fuzzer and fuzzing with it
+
+This task is to be completed in a Jupyter notebook.
+
+You can access an online notebook following this link: https://notebooks.rahtiapp.fi/welcome TODO?
```

Comment:
There is a risk that fuzzing causes a lot of resource use on rahtiapp or that the underlying machine might be too slow.